conf/views/confirmation/validationErrors.njk

```diff
@@ -0,0 +1,57 @@
+{% extends "includes/layout.njk" %}
+{% from "govuk/components/table/macro.njk"         import govukTable %}
+{% from "govuk/components/back-link/macro.njk"     import govukBackLink %}
+{% from "macros/title.njk"                         import title %}
+
+{% block pageTitle %}
+  {{ title(messages("confirmation.validationError.title")) }}
+{% endblock %}
+
+{% block mainContent %}
+
+  {{ govukBackLink({
+    text: messages("site.back")
+  }) }}
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">
+          {{ messages("confirmation.validationError.title") }}
+        </h1>
+
+        <p class="govuk-body">{{ messages("confirmation.validationError.heading") }}</p>
+
+        <h2 class="govuk-heading-m">
+          {{ messages("confirmation.validationError.info") }}
+        </h2>
+
+        <ol class="govuk-list govuk-list--number">
+          <li class="govuk-body"><a href="#" onclick="window.print();return false;">{{ messages("confirmation.validationError.info.p1") }}</a></li>
```

Comment:
Use an event listener instead. There's an example in dac.js in disclose frontend.